packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -759,12 +760,59 @@ export class AreaSeries extends CartesianSeries<AreaSeriesNodeDataContext> {
             lineDashOffset: this.lineDashOffset,
             strokeOpacity: this.strokeOpacity,
             shadow: this.shadow,
-            strokeWidth: () => this.getStrokeWidth(this.strokeWidth),
+            strokeWidth: this.getStrokeWidth(this.strokeWidth),
         };
 
         areaAnimateReadyUpdate({ contextData, paths, styles });
     }
 
+    animateReadyResize({ contextData, markerSelections, labelSelections, paths }: AreaAnimationData) {
+        this.ctx.animationManager?.reset();
```

Comment:
Just noticed we are doing this in a few `animateReadyResize()` cases, looks a bit suspicious? Maybe we could sync up this afternoon?